plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx

```diff
@@ -22,34 +24,97 @@ export const ProductGalleryThumbnailsBlockSettings = ( {
 	attributes,
 	setAttributes,
 }: ProductGalleryThumbnailsSettingsProps ) => {
-	const { thumbnailSize } = attributes;
+	const { thumbnailSize, aspectRatio } = attributes;
+
+	const aspectRatioOptions = [
+		{
+			value: '',
+			label: __( 'Select Aspect Ratio', 'woocommerce' ),
+			disabled: true,
+		},
+		{
+			value: 'auto',
+			label: __( 'Original', 'woocommerce' ),
+		},
+		{
+			value: '1',
+			label: __( 'Square - 1:1', 'woocommerce' ),
+		},
+		{
+			value: '4/3',
+			label: __( 'Standard - 4:3', 'woocommerce' ),
+		},
+		{
+			value: '3/4',
+			label: __( 'Portrait - 3:4', 'woocommerce' ),
+		},
+		{
+			value: '3/2',
+			label: __( 'Classic - 3:2', 'woocommerce' ),
+		},
+		{
+			value: '2/3',
+			label: __( 'Classic Portrait - 2:3', 'woocommerce' ),
+		},
+		{
+			value: '16/9',
+			label: __( 'Wide - 16:9', 'woocommerce' ),
+		},
+		{
+			value: '9/16',
+			label: __( 'Tall - 9:16', 'woocommerce' ),
+		},
+		{
+			value: 'custom',
+			label: __( 'Custom', 'woocommerce' ),
+			disabled: true,
+			hidden: true,
+		},
+	];
 
 	return (
-		<UnitControl
-			label={ __( 'Thumbnail Size', 'woocommerce' ) }
-			value={ thumbnailSize }
-			onChange={ ( value: string | undefined ) => {
-				const numberValue = Number(
-					value?.replace( '%', '' ) || defaultValue
-				);
-				const validated = Math.min(
-					Math.max( numberValue, minValue ),
-					maxValue
-				);
-				setAttributes( {
-					thumbnailSize: validated + '%',
-				} );
-			} }
-			units={ [ { value: '%', label: '%' } ] }
-			min={ minValue }
-			max={ maxValue }
-			step={ 1 }
-			size="default"
-			__next36pxDefaultSize
-			help={ __(
-				'Choose the size of each thumbnail in respect to the product image. If thumbnails container size gets bigger than the product image, thumbnails will turn to slider.',
-				'woocommerce'
-			) }
-		/>
+		<PanelBody className="wc-block-product-gallery-thumbnails__tools-panel">
+			<UnitControl
+				label={ __( 'Thumbnail Size', 'woocommerce' ) }
+				value={ thumbnailSize }
+				onChange={ ( value: string | undefined ) => {
+					const numberValue = Number(
+						value?.replace( '%', '' ) || defaultValue
+					);
+					const validated = Math.min(
+						Math.max( numberValue, minValue ),
+						maxValue
+					);
+					setAttributes( {
+						thumbnailSize: validated + '%',
+					} );
+				} }
+				units={ [ { value: '%', label: '%' } ] }
+				min={ minValue }
+				max={ maxValue }
+				step={ 1 }
+				size="default"
+				__next36pxDefaultSize
+				help={ __(
+					'Choose the size of each thumbnail in respect to the product image. If thumbnails container size gets bigger than the product image, thumbnails will turn to slider.',
+					'woocommerce'
+				) }
+			/>
+			<SelectControl
+				multiple={ false }
```

Comment:
Nitpick, but please add `__next40pxDefaultSize` to align the styling of the component in Inspector Controls

```suggestion
			<SelectControl
			    __next40pxDefaultSize
				multiple={ false }
```

I know it's not commonly applied but I'm trying to spread knowledge about it 😅 

Without | With `__next40pxDefaultSize`
-- | --
<img width="281" alt="image" src="https://github.com/user-attachments/assets/c1c1a2fb-a2c6-4652-a2ef-351f9d073a11" /> | <img width="282" alt="image" src="https://github.com/user-attachments/assets/cc3f1cef-9d44-49a1-9823-3117b8a16a66" />

Reply:
Thanks for this suggestion ! 
should i also change this [ref](https://github.com/woocommerce/woocommerce/blob/5082e0a0ae8963d1e1d0c7285ce7cd0db7ae7897/plugins/woocommerce/client/blocks/assets/js/blocks/product-gallery/inner-blocks/product-gallery-thumbnails/block-settings/index.tsx#L48) to __next40pxDefaultSize ?